recordatorio_estudio.py

```diff
@@ -17,7 +17,13 @@ def center_window(window, width_percentage=0.25, height_percentage=0.25):
 def on_yes():
     def check_confirmation(event=None):
         if entry.get() == settings["message"][1]:
-            subprocess.run(rf'"{settings["exe_path"]}" --launch-product={settings["launch_product"]} --launch-patchline=live', shell=True)
+            cmd = rf'"{settings["exe_path"]}"'
+
+            if settings["extra_args"] != "":
```

Comment:
it's better to change this to `"extra_args" in settings` because not having any extra_args in the settings could be a valid state and in those cases this will return an Error.